apyros/metaopen.py

```diff
@@ -0,0 +1,20 @@
+"""
+   only wrapper for ZIP files
+"""
+
+import os
+import zipfile
+import StringIO
+
+
+def metaopen(filename, mode='r'):
+    """Wrapper for opening files optionally from ZIP"""
+    assert mode in ['r', 'rb'], mode  # just to know what modes we use
+    if '.zip' in filename:
+        data = zipfile.ZipFile(os.path.dirname(filename)).read(os.path.basename(filename))
+        print len(data)
+        return StringIO.StringIO(data)
```

Comment:
Let's use [ZipFile.open()](https://docs.python.org/2/library/zipfile.html?highlight=zipfile#zipfile.ZipFile.open) here, ok?
Also, should the debugging `print len(data)` really stay there?